docs/changelog.rst

```diff
@@ -27,6 +27,9 @@ Fixes:
 * Remove the ``beatport`` plugin. `Beatport`_ has shut off public access to
   their API and denied our request for an account. We have not heard from the
   company since 2013, so we are assuming access will not be restored.
+* Fix tracktotal being stored incorrectly for multi-disc releases where the
+  individual discs have a different number of tracks and per_disc_numbering
+  is enabled.
```

Comment:
This deserves a little more emphasis: we should warn people that `tracktotal` is no longer a valid album field.